maintainers/maintainer-list.nix

```diff
@@ -3822,6 +3822,18 @@
     githubId = 6884440;
     name = "Ding Xiang Fei";
   };
+  d-xo = {
+    email = "hi@d-xo.org";
+    github = "d-xo";
+    githubId = 6689924;
+    name = "David Terry";
+  };
+  dylanmtaylor = {
+    email = "dylan@dylanmtaylor.com";
+    github = "dylamtaylor";
```

Comment:
```suggestion
    github = "dylanmtaylor";
```

Reply:
cherry-picked the fix in https://github.com/NixOS/nixpkgs/pull/225163/commits/8add8664729470f2472ee91e26919f17935b327d